CMakeLists.txt

```diff
@@ -125,7 +125,7 @@ option(EPROSIMA_BUILD "Activate internal building" OFF)
 ###############################################################################
 # Load external eProsima projects.
 ###############################################################################
-set(FASTDDS_MIN_VERSION "3.0.0")
+set(FASTDDS_MIN_VERSION "3.0.1")
```

Comment:
I doubt here whether we should keep `3.0.0`

Reply:
It is not part of the bumping process.
I guess we updated it due to the major version, but I would keep it as it was (`3.0.0`).